target/dita/regions/regions.dita

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="utf-8"?>
+&lt;!DOCTYPE topic PUBLIC "-//OASIS//DTD DITA Topic//EN" "topic.dtd"&gt;
+<topic id="links_1">
+ <title>
+  Regions
+ </title>
+ <body>
+  <imagemap>
+   <image href="./Content/Images/world-map.gif"/>
+   <area>
+    <shape>
+     poly
+    </shape>
+    <coords>
+     550,211,522,231,515,263,540,275,564,269,561,231,555,214
+    </coords>
+    <xref format="dita" href="../Britain/Britain1.html"/>
```

Comment:
this XREF will fail, since there isn't an html file at `../Britain/Britain1.html`.  Instead, we'll just link to the current file, so it's `./regions.dita`.

We'll create these country folders (and files) in the next issue.